### ISSUE TITLE
Documentation?

### ISSUE THREAD

**Issue by TermitCG** [addisonElliott/pypdn#12]
Is there any documentation of this library?

**Comment by addisonElliott** [comment]
No, there isn't any documentation. I was going to wait and add documentation once this library gained some more popularity/traction.

The example given in the README is a good starting point. Besides that, you can look at the tests as well.

If you have a specific question, let me know and I *might* be able to help.

**Comment by eqprog** [comment]
Looking for a way to *write* pdn files with multiple layers. Is this possible? If yes, could you give an example?

**Comment by addisonElliott** [comment]
Looks like I never got around to it: https://github.com/addisonElliott/pypdn/blob/master/pypdn/reader.py#L11

Sorry. Hopefully some of the infrastructure is in place for writing PDN files that you could figure something out. As always, PRs are welcome.
